renderer/src/hooks/StationWallet.tsx

```diff
@@ -138,10 +143,10 @@ interface SplitTransactions {
 }
 
 const splitWalletTransactions = (transactions: (FILTransaction|FILTransactionProcessing)[]): SplitTransactions => {
-  const processing: FILTransactionProcessing | null =
-  transactions.find(tx => tx.status === 'processing') as FILTransactionProcessing
+  const processing: FILTransactionProcessing | undefined =
+  transactions.find(isFILTransactionProcessing)
   const confirmed: FILTransaction[] =
-  transactions.filter(tx => tx.status !== 'processing') as FILTransaction[]
+  transactions.filter(isFILTransactionConfirmed)
```

Comment:
Ditto. I think the following should work too.

```ts
const confirmed = transactions.filter(isFILTransactionConfirmed)
```

Reply:
(Assuming you like this terse version more than the original.)

Reply:
I do :)